client/src/pages/instructor/InstructorLayout.js

```diff
@@ -1,10 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Container, Row } from 'react-bootstrap';
 import { Outlet } from 'react-router-dom';
+import Footer from '../../components/Footer';
+import Header from '../../components/instructor/Header';
+import SideNavigationBar from '../../components/instructor/SideNavigationBar';
 
 function InstructorLayout() {
+  const [showSidebar, setShowSidebar] = useState(false);
+
   return (
     <>
-      <Outlet></Outlet>
+      <Header showSidebar={showSidebar} setShowSidebar={setShowSidebar} />
+      <Container fluid>
+        <Row id='instructor-dashboard'>
+          <SideNavigationBar showSidebar={showSidebar} setShowSidebar={setShowSidebar} />
+          <Outlet></Outlet>
+        </Row>
+      </Container>
+      <Footer />
```

Comment:
Before you make the next commit, please read about js and React coding conventions and format the code.

Reply:
code formatted